### PR TITLE
Blog balena and FlowFuse

### PR DESCRIPTION
**Pull request by gdziuba**
## Related Issue(s)

close https://github.com/FlowFuse/website/issues/1591

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/blog/2024/01/how-to-deploy-node-red-with-flowfuse-to-balenacloud.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+date: 2024-01-29
+authors: ["grey-dziuba"]
+image: 
+tags:
+    - posts
+    - flowfuse
+    - how-to
+---
+
+In a recent webinar with balena, we explored the dynamic capabilities of deploying FlowFuse to a fleet of devices using balena Cloud. This blog post serves as a practical guide to replicate that process, specifically tailored for those aiming to streamline their deployment of FlowFuse in an efficient and user-friendly manner.
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/how-to-deploy-node-red-with-flowfuse-to-balenacloud.md:14]:
```suggestion
In a [recent webinar with balena](https://flowfuse.com/webinars/2024/balena/), we explored the dynamic capabilities of deploying FlowFuse to a fleet of devices using balena Cloud. This blog post serves as a practical guide to replicate that process, specifically tailored for those aiming to streamline their deployment of FlowFuse in an efficient and user-friendly manner.
```

**Review by ZJvandeWeg — APPROVED** [review]

File: src/blog/2024/01/how-to-deploy-node-red-with-flowfuse-to-balenacloud.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: Step-by-Step Guide to Deploying Node-RED with FlowFuse in balena Cloud
+subtitle: Fleet management made easier with FlowFuse and balena.
+description: Fleet management made easier with FlowFuse and balena.
+date: 2024-01-29
+authors: ["grey-dziuba"]
+image:
```
Comment (ZJvandeWeg) [inline src/blog/2024/01/how-to-deploy-node-red-with-flowfuse-to-balenacloud.md:7]: @gdziuba Do you want a tile? Or no?

Comment (gdziuba) [reply src/blog/2024/01/how-to-deploy-node-red-with-flowfuse-to-balenacloud.md:7]: @ZJvandeWeg  Yes.  https://github.com/FlowFuse/website/issues/1591

File: src/blog/2024/01/how-to-deploy-node-red-with-flowfuse-to-balenacloud.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+title: Step-by-Step Guide to Deploying Node-RED with FlowFuse in balena Cloud
+subtitle: Fleet management made easier with FlowFuse and balena.
+description: Fleet management made easier with FlowFuse and balena.
+date: 2024-01-29
```
Comment (Yndira-FlowForge) [inline src/blog/2024/01/how-to-deploy-node-red-with-flowfuse-to-balenacloud.md:5]:
```suggestion
date: 2024-01-30
```

**Pull request merged**